tfjs-backend-webgpu/src/kernel_utils/kernel_funcs_utils.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
`2020` -> `2021`. And change it in all places. Thanks.

Reply:
fixed